From 44c333f8c1533930585414cb8575c701beb20a0b Mon Sep 17 00:00:00 2001
From: Saul Wold <sgw@linux.intel.com>
Date: Thu, 22 Feb 2018 12:38:17 -0800
Subject: [PATCH 553/692] REVERTME: usb/gadget: fix x86_mask -> x86_stepping

The upstream commit 325cbb04dc61692ba01445c34fd551ad91df9e6d changed
the cpu_data member name from x86_mask to x86_stepping so patch the
'dwc3 setup highspeed to USB3.0 on bxtp platform' patch that was not
known to the original author.

Signed-off-by: Saul Wold <sgw@linux.intel.com>
---
 drivers/usb/dwc3/gadget.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/usb/dwc3/gadget.c b/drivers/usb/dwc3/gadget.c
index e9baac9..dce018f 100644
--- a/drivers/usb/dwc3/gadget.c
+++ b/drivers/usb/dwc3/gadget.c
@@ -1858,8 +1858,8 @@ static inline bool platform_is_bxtp(void)
 {
 #ifdef CONFIG_X86_64
 	if ((boot_cpu_data.x86_model == 0x5c)
-		&& (boot_cpu_data.x86_mask >= 0x8)
-		&& (boot_cpu_data.x86_mask <= 0xf))
+		&& (boot_cpu_data.x86_stepping >= 0x8)
+		&& (boot_cpu_data.x86_stepping <= 0xf))
 		return true;
 #endif
 	return false;
-- 
2.7.4

